css/font-awesome/css/font-awesome.css

```diff
@@ -0,0 +1,2199 @@
+/*!
+ *  Font Awesome 4.6.3 by @davegandy - http://fontawesome.io - @fontawesome
+ *  License - http://fontawesome.io/license (Font: SIL OFL 1.1, CSS: MIT License)
+ */
```

Comment:
I would like to avoid putting binaries in the version control, how about adding it as a npm dependency?